cttportfolio.css

```diff
@@ -1,470 +1,874 @@
+
+
 * {
 	box-sizing: border-box;
 }
-html{
-	font-size:62.5%;
-	font-family:'Montserrat', sans-serif;
-	color:rgba(4,8,9,1);
+
+html {
+	font-size: 62.5%;
+	font-family: 'Montserrat', sans-serif;
+	color: rgba(4, 8, 9, 1);
```

Comment:
This doesn't need to be `rgba()`, an alpha of `1` is the same as just using `rgb()`.